plugins/inputs/nats_consumer/nats_consumer.go

```diff
@@ -34,6 +34,7 @@ type NatsConsumer struct {
 	Credentials            string          `toml:"credentials"`
 	NkeySeed               string          `toml:"nkey_seed"`
 	JsSubjects             []string        `toml:"jetstream_subjects"`
+	JsStream               string          `toml:"jetstream_stream"`
```

Comment:
Is there a chance we want to consume multiple streams in the future or is NATS limited to one stream?

Reply:
Ok, then keep as is. Thanks for the explanation!